packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerCard/AddPrayerCardConnected.js

```diff
@@ -0,0 +1,20 @@
+import React, { memo } from 'react';
+import { Query } from 'react-apollo';
+import { get } from 'lodash';
+import getProfilePic from '../../getProfilePic';
+import AddPrayerCard from './AddPrayerCard';
+
+const AddPrayerCardConnected = memo(({ ...props }) => (
+  <Query query={getProfilePic} fetchPolicy={'cache-only'}>
+    {({ data }) => (
+      <AddPrayerCard
+        imgSrc={get(data, 'currentUser.profile.photo', { uri: '' })}
```

Comment:
good use of `get` 👍 

I do believe `{ uri: '' }` would be considered invalid input though...might considering returning `null` instead.